### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Manya67**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,126 @@
...
+- [Configuration](#configuration)
+- [API Documentation](#api-documentation)
+- [Support](#support)
+
+## Installation
+
+To get started, install the Kursaha SDK via npm:
+
+```bash
+npm install kursaha-sdk
```
Comment (Nishant-Pathak) [inline README.md]: --save

File: README.md [hunk 2]
```diff
@@ -1 +1,126 @@
...
+```
+
+## Getting Started
+
+1. **Initialize the SDK**
```
Comment (Nishant-Pathak) [inline README.md:27]: add import line too as it is complex

File: README.md [hunk 3]
```diff
@@ -1 +1,126 @@
...
+
+## Methods
+
+### Signal Start Event
+
+Send a signal to start an event flow. This method allows you to track various events within your application.
+
+```javascript
+const signal = {
+  customerId: 'unique-emitter-id',
```
Comment (Nishant-Pathak) [inline README.md]: unique-**customer**-id

File: README.md [hunk 4]
```diff
@@ -1 +1,126 @@
...
+  eventType: 'step-node-id',
```
Comment (Nishant-Pathak) [inline README.md]: event-type in please of step-node-id

File: package.json [hunk 5]
```diff
@@ -13,6 +13,12 @@
   "peerDependencies": {
     "react": ">=14"
   },
+  "exports": {
```
Comment (Nishant-Pathak) [inline package.json]: why we need this ?

Comment (Manya67) [reply package.json]: to make import straightforward

**Review by Nishant-Pathak — APPROVED** [review]

**Pull request merged**